### PR TITLE
Remove PostgreSQL on AWS for SAP deployments

### PR DESCRIPTION
**Pull request by MarkvanMents**
[MCDEP-1096]

### CODE REVIEW

File: content/en/docs/developerportal/deploy/sap-cloud-platform.md [hunk 1]
```diff
@@ -585,29 +581,25 @@ Only use unsupported environment variables if you know exactly what you are doin
...
 
-Mendix needs access to a relational database back end and can run using different types of database. For deployment to SAP BTP, you have the choice of PostgreSQL or SAP HANA.
+Mendix needs access to a relational database back end and can run using different types of database. For deployment to SAP BTP, you have the choice of PostgreSQL Hyperscaler or SAP HANA.
 
-### 8.1 Running Mendix on PostgreSQL
+### 8.1 Running Mendix on PostgreSQL on SAP BTP{#sap-hyperscaler}
 
 {{% alert color="warning" %}}
-SAP have deprecated the use of PostgreSQL databases bound via the marketplace. This means that SAP accounts created recently will not be able to directly bind PostgreSQL databases and you will have to use [PostgreSQL on SAP BTP](#sap-hyperscaler), [SAP HANA](#sap-hana), or [AWS RDS PostgreSQL](#aws-rds) databases.
+SAP have deprecated the use of PostgreSQL databases bound via the marketplace. This means that, if you need a PostgreSQL database, you will have to use PostgreSQL on SAP BTP as your database. Alternatively, you can use [SAP HANA](#sap-hana).
```
Comment (gauravkrshah) [inline content/en/docs/developerportal/deploy/sap-cloud-platform.md]: Didn't understand this line. Postgres hyperscaler is also on SAP Martketplace.

**Review by gauravkrshah — COMMENTED** [review]
All looks good. Except that marketplace line.

File: content/en/docs/developerportal/deploy/sap-cloud-platform.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/developerportal/deploy/sap-cloud-platform.md]: Hi @gauravkrshah 
It was left over from the previous warning, which said that they were deprecated. Would this do as an update?
```suggestion
SAP have removed some of the PostgreSQL databases available in their marketplace. If you need a PostgreSQL database, you will have to use PostgreSQL on SAP BTP as your database. This requires some extra configuration which is described below. Alternatively, you can use [SAP HANA](#sap-hana).
```

File: content/en/docs/developerportal/deploy/sap-cloud-platform.md [hunk 2]
```diff
@@ -585,29 +581,25 @@ Only use unsupported environment variables if you know exactly what you are doin
 
 ## 8 Databases in SAP BTP{#databases}
 
-Mendix needs access to a relational database back end and can run using different types of database. For deployment to SAP BTP, you have the choice of PostgreSQL or SAP HANA.
+Mendix needs access to a relational database back end and can run using different types of database. For deployment to SAP BTP, you have the choice of PostgreSQL Hyperscaler or SAP HANA.
 
-### 8.1 Running Mendix on PostgreSQL
+### 8.1 Running Mendix on PostgreSQL on SAP BTP{#sap-hyperscaler}
 
 {{% alert color="warning" %}}
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-cloud-platform.md]: Perhaps this is only information, now that customers don't have a choice of PostgreSQL?
```suggestion
{{% alert color="info" %}}
```

**Review by gauravkrshah — APPROVED** [review]
Looks Good. Thanks!

File: content/en/docs/developerportal/deploy/sap-cloud-platform.md [hunk 3]
```diff
@@ -585,29 +581,25 @@ Only use unsupported environment variables if you know exactly what you are doin
...
 
-Mendix needs access to a relational database back end and can run using different types of database. For deployment to SAP BTP, you have the choice of PostgreSQL or SAP HANA.
+Mendix needs access to a relational database back end and can run using different types of database. For deployment to SAP BTP, you have the choice of PostgreSQL, Hyperscaler Option or SAP HANA.
 
-### 8.1 Running Mendix on PostgreSQL
+### 8.1 Running Mendix on PostgreSQL, Hyperscaler Option{#sap-hyperscaler}
 
 {{% alert color="warning" %}}
-SAP have deprecated the use of PostgreSQL databases bound via the marketplace. This means that SAP accounts created recently will not be able to directly bind PostgreSQL databases and you will have to use [PostgreSQL on SAP BTP](#sap-hyperscaler), [SAP HANA](#sap-hana), or [AWS RDS PostgreSQL](#aws-rds) databases.
+SAP have deprecated the use of PostgreSQL databases bound via the marketplace. This means that, if you need a PostgreSQL database, you will have to use PostgreSQL, Hyperscaler Option as your database. Alternatively, you can use [SAP HANA](#sap-hana).
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-cloud-platform.md]:
```suggestion
SAP have removed some of the PostgreSQL databases available in their marketplace. If you need a PostgreSQL database, you will have to use PostgreSQL, Hyperscaler Option as your database. This requires some extra configuration which is described below. Alternatively, you can use [SAP HANA](#sap-hana).
```

**Pull request merged**